packages/components/src/presentation-components/instance-filter-builder/PresentationInstanceFilterDialog.scss

```diff
@@ -4,40 +4,43 @@
 *--------------------------------------------------------------------------------------------*/
 
 .presentation-instance-filter-dialog {
-  min-width: 50vw;
+  .presentation-instance-filter-dialog-content-container {
+    min-height: calc(var(--iui-size-2xl) * 5 + var( --iui-size-xs));
+    min-width: calc(var(--iui-size-2xl) * 8);
 
-  .presentation-instance-filter-title {
-    margin-bottom: 0;
-    padding: 6px;
-  }
+    .presentation-instance-filter-title {
+      margin-bottom: 0;
+      padding: 6px;
+    }
 
-  .presentation-instance-filter-content {
-    max-height: 80vh;
-    overflow: hidden;
+    .presentation-instance-filter-content {
+      max-height: 80vh;
```

Comment:
Can you check if this is still needed? If I remember correctly this was added to make sure that Dialog does not grow out off screen when adding new filter rules. However, Dialog is resizable now and it has static size.

Reply:
removed